src/common/container/cuckoo_map.cpp

```diff
@@ -83,6 +87,18 @@ CUCKOO_MAP_TEMPLATE_ARGUMENTS
 CUCKOO_MAP_ITERATOR_TYPE
 CUCKOO_MAP_TYPE::GetIterator() { return cuckoo_map.lock_table(); }
 
+CUCKOO_MAP_TEMPLATE_ARGUMENTS
```

Comment:
Why do we need this const decorator?

Reply:
This is because the CUKOO_MAP_ITERATOR_TYPE needs template typenames for KeyType, ValueType, HashType and PredType.